diff --git a/params/config.go b/params/config.go
index bd5c40b..f56aa79 100644
--- a/params/config.go
+++ b/params/config.go
@@ -1122,7 +1122,7 @@ func (c *ChainConfig) Rules(num *big.Int, isMerge bool, timestamp uint64) Rules
 		IsOptimismBedrock:  isMerge && c.IsOptimismBedrock(num),
 		IsOptimismRegolith: isMerge && c.IsOptimismRegolith(timestamp),
 		IsOptimismCanyon:   isMerge && c.IsOptimismCanyon(timestamp),
-		IsOptimismFjord:    isMerge && c.IsOptimismFjord(timestamp),
+		IsOptimismFjord:    c.IsFjord(timestamp),
 		IsOptimismGranite:  isMerge && c.IsOptimismGranite(timestamp),
 	}
 }
